sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2AlterTableCommands.scala

```diff
@@ -144,7 +144,7 @@ case class ReplaceColumns(
     require(table.resolved)
     val deleteChanges = table.schema.fieldNames.map { name =>
       // REPLACE COLUMN should require column to exist
-      TableChange.deleteColumn(Array(name), ifExists = false)
```

Comment:
Sorry I'm a bit confused. named parameter has been widely used in the Spark codebase, what's wrong with it?

Reply:
Yeah, named parameter is right. But there is a compilation error when k8s integration test. Please see PR description for more information. And you can exec this command to check it again
```shell
build/sbt -Pkubernetes -Pkubernetes-integration-tests -Dtest.exclude.tags=r -Dspark.kubernetes.test.imageRepo=kubespark "kubernetes-integration-tests/test"
```

Reply:
Do you know why the issue only happens in this place?

Reply:
Because deleteColumn#TableChange is `Java` file. I replace named parameter with comment, because it's used in Apache Spark. https://github.com/apache/spark/blob/ad81ba839713374b2ef9ea2f0f18e83578584188/core/src/main/scala/org/apache/spark/deploy/ExternalShuffleService.scala#L119